config/ImathConfig.h.in

```diff
@@ -50,4 +50,13 @@
                              (uint32_t(IMATH_VERSION_MINOR) << 16) | \
                              (uint32_t(IMATH_VERSION_PATCH) <<  8))
 
+//
+// Constexpr C++14 conditional definition
+//
+#if(__cplusplus >= 201402L)
```

Comment:
Older versions of MSVS were not good about setting this, and so you will end up with C++14 compliant MSVS versions that will not constexpr these even though they can. I would recommend the following test instead:

    #if (__cplusplus >= 201402L) || (defined(_MSC_VER) && _MSC_VER >= 1914)